model/feature-flag/registry.yaml

```diff
@@ -9,26 +9,45 @@ groups:
       - id: feature_flag.key
         type: string
         stability: experimental
-        brief: The unique identifier of the feature flag.
+        brief: The lookup key of the feature flag.
         examples: ["logo-color"]
-      - id: feature_flag.provider_name
+      - id: feature_flag.provider.id
         type: string
         stability: experimental
-        brief: The name of the service provider that performs the flag evaluation.
+        brief: Any string which identifies the feature flag provider being used such as its name.
         examples: ["Flag Manager"]
       - id: feature_flag.variant
         type: string
         stability: experimental
         examples: ["red", "true", "on"]
         brief: >
-          SHOULD be a semantic identifier for a value. If one is unavailable, a
-          stringified version of the value can be used.
+          A semantic identifier for an evaluated flag value.
         note: |-
           A semantic identifier, commonly referred to as a variant, provides a means
           for referring to a value without including the value itself. This can
           provide additional context for understanding the meaning behind a value.
           For example, the variant `red` maybe be used for the value `#c05543`.
-
-          A stringified version of the value can be used in situations where a
-          semantic identifier is unavailable. String representation of the value
-          should be determined by the implementer.
+      - id: feature_flag.value
+        type: string
```

Comment:
Does SemCon support an `any` type?

Reply:
We could consider moving the flag value to be a body field, which would allow us to use `unknown` for its type. I'm still not clear when it is appropriate to use a body field vs an attribute. Maybe @lmolkova can make a suggestion.